src/Menus/Scheduler.json

```diff
@@ -35,6 +35,16 @@
                 "title": "Manage Terms",
                 "url": "module=Scheduler,action=listTerms",
                 "description": "Seasons are applied to in relation to Terms. Manage Terms here."
+            },
```

Comment:
Worth moving these into a separate module? Scheduler is getting quite chonk

Reply:
ye...maybe. but not now (mainly cause i can't really be bothered to) because all changing pemissions and fun :)